prow/overlays/smaug/ghproxy.yaml

```diff
@@ -61,11 +61,11 @@ spec:
               mountPath: /cache
           resources:
             requests:
-              memory: "64Mi"
-              cpu: "2m"
+              memory: "128Mi"
+              cpu: "10m"
             limits:
-              memory: "64Mi"
-              cpu: "2m"
+              memory: "256Mi"
```

Comment:
This switches from guaranteed to burstable QoS. Do we want that? Or should we keep limits == requests so that the pod doesn't get unexpectedly killed?

Reply:
I agree with you, however I'd say let's keep it this way for now and see the actual consumption. Once we know it, we can match limits == requests afterwards to achieve guaranteed QoS. WDYT?

Reply:
Makes sense. :shipit: